de-lib/src/main/java/org/iplantc/de/apps/integration/client/view/propertyEditors/widgets/SelectionItemPropertyEditor.java

```diff
@@ -239,9 +239,9 @@ public void setSuppressEvent(boolean suppressEventFire) {
     ColumnModel<SelectionItem> createColumnModel() {
         List<ColumnConfig<SelectionItem, ?>> list = Lists.newArrayList();
         SelectionItemProperties props = GWT.create(SelectionItemProperties.class);
-        displayCol = new ColumnConfig<>(props.display(), 120, labels.singleSelectDisplayColumnHeader());
-        nameCol = new ColumnConfig<>(props.name(), 150, labels.singleSelectNameColumnHeader());
-        valueCol = new ColumnConfig<>(props.value(), 150, labels.singleSelectValueColumnHeader());
+        displayCol = new ColumnConfig<>(props.display(), 120, appearance.singleSelectDisplayColumnHeader());
+        nameCol = new ColumnConfig<>(props.name(), 150, appearance.singleSelectNameColumnHeader());
+        valueCol = new ColumnConfig<>(props.value(), 150, appearance.singleSelectValueColumnHeader());
```

Comment:
One of these days, maybe these column widths should also go into the appearance?
It looks like we do put column widths into appearances for some things.